IlliniBlockchain/buoyant
# 🍎 Initialize Instruction

To run/test:
1. Compile and deploy program
2. Copy program id into constant variable at top of `initialize.js`
3. Run `initialize.js` - takes 2 params: amount and duration
```
node initialize.js [AMOUNT] [DURATION]
```
If successful should print out link to testnet and count of the type of subscription.
Example:
1. https://explorer.solana.com/tx/iMpcn9zNU7hiPhscQdtifrVDmcrPEnYnS6ZZhriWnZpeE1Q9oFvj1DBPENPToJndYE2PeV8RJDFhLLfohQZ5TFm?cluster=devnet
2. https://explorer.solana.com/tx/3atYi7XW9MnKskRwuXpnHWjyBd6kKvDgJvX17XciWQrLZMAVyqo3UTHCJZMy11Dr3ZxGZ2tjednjpAEVEtXQCUK5?cluster=devnet

Note: in my initialize script I essentially jun the thing twice to ensure the counter is incremented properly.

Closes #4
